mne/artifact_detection.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Nov 19 09:40:37 2019
+
+@author: an512
```

Comment:
This is not kosher. Put this file under `mne/preprocessing/` and then format the header like the other files.

Reply:
I thought that these scripts were temporary. Fixing it!